membership-attribute-service/app/services/AccountDetailsFromZuora.scala

```diff
@@ -224,7 +224,7 @@ class AccountDetailsFromZuora(
       case Product.Digipack => requestedProductType == "Digipack" || requestedProductTypeIsContentSubscription
       case Product.SupporterPlus => requestedProductType == "SupporterPlus" || requestedProductTypeIsContentSubscription
       case Product.TierThree => requestedProductType == "TierThree" || requestedProductTypeIsContentSubscription
-      case Product.GuardianLight => requestedProductType == "GuardianLight" || requestedProductTypeIsContentSubscription
+      case Product.GuardianAdLite => requestedProductType == "GuardianAdLite" || requestedProductTypeIsContentSubscription
```

Comment:
according to ProductFamily.scala ad lite is not a content subscription
https://github.com/guardian/members-data-api/pull/1111/files#diff-8434c9ffdcbb8fc26673d1f84012690fee7a77abf6ab033c73335a7b03a32f1aR36
I'm not 100% sure the significance of this check, but we should be consistent.

Reply:
seems like it's used in MMA for things like whether to display a delivery address change
https://github.com/guardian/manage-frontend/blob/a8c65fe9d719b5bef22a6931f3e9477ebeb54a73/client/components/mma/delivery/records/DeliveryAddressStep.tsx#L612
So I would say this shouldn't be returned for ContentSubscription
```suggestion
      case Product.GuardianAdLite => requestedProductType == "GuardianAdLite"
```